Tests/SentryTests/SentryHubTests.swift

```diff
@@ -291,6 +291,66 @@ class SentryHubTests: XCTestCase {
         let span = hub.startTransaction(name: fixture.transactionName, operation: fixture.transactionOperation)
         XCTAssertEqual(span.context.sampled, .no)
     }
+
+#if os(iOS)
```

Comment:
?
```suggestion
#if os(iOS) || os(tvOS)
```

Reply:
This doesn't work on tvOS because the relevant kernel APIs are not available